molgenis/molgenis-emx2
# fix: negotiator error message and switch to new negotiator API endpoint

fixes: #3893; #4045

What are the main changes you did:
- Display an error message in the checkout when negotiator is unavailable.

how to test:
- Go to the directory app.
- Add a collection to the negotiator selection.
- Press "Request" and see error message.
